cmake/std/atdm/mutrino/tweaks/GNU-DEBUG-SERIAL.cmake

```diff
@@ -0,0 +1,5 @@
+# Disable test that times out after 10 minutes (#2455)
```

Comment:
Same for `GNU` tweak files.  The file `mutrino/environment.sh` does not even support a `GNU` build.  Should all of these files be removed as well?